### PR TITLE
fe/home

### PR DESCRIPTION
**Pull request by M-Faheem1**

### CODE REVIEW

File: _variables.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$primary: #b624ff;
+$font: 'DM Sans', 'sans-serif';
+$myfont: 'Poppins', 'sans-serif';
```
Comment (M-Faheem1) [inline _variables.scss]: myfont->secondary_font

File: CogentLabs.html [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" 
+  rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" 
+  crossorigin="anonymous">
+  <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" 
+  integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" 
+  crossorigin="anonymous"></script>
+  <title>Cogent Labs</title>
+</head>
+<body>
+  <nav class="navbar site">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: remove site class.

File: CogentLabs.html [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+        </mask>
+        <g mask="url(#mask0_828_542)">
+        <rect x="-5.19043" y="-21.6641" width="186.86" height="89.9696" rx="6" fill="black"/>
+        </g>
+      </svg>
+    </div>
+    <ul>
+      <li>
+        <a href="#services" target="_self">Services</a>
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: remove target

File: CogentLabs.html [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+        <a href="#about-us" target="_self">About Us</a>
+      </li>
+      <li>
+        <a href="#career" target="_self">Career</a>
+      </li>
+      <li>
+        <a href="#portfolio" target="_self">Blog</a>
+      </li>
+    </ul>
+    <button class="btn btn-primary ">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: remove extra white space.

File: CogentLabs.html [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+      Book a Consultation 
+    </button>
+  </nav>
+  <div class="container">
+    <div class="consultaion d-flex flex-column">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: consultaion -> consultation

File: CogentLabs.html [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+      <img src="images/image-3.png">
+    </div>
+    <div class="image item-4">
+      <img src="images/image-4.png">
+    </div>
+    <div class="image item-5">
+      <img src="images/image-5.png">
+    </div>
+  </div>
+  <div class="container mt-5 services" id="services">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: move mt-5 styling to style.scss.

File: CogentLabs.html [hunk 7]
```diff
@@ -0,0 +1,440 @@
...
+    <h3>Our Services</h3>
+    <h4>
+      Services We Provide For Our
+      <span>Clients Worldwide</span> 
+    </h4>
+    <div class="card-listing mt-lg-5">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: remove `mt-lg-5`.

File: CogentLabs.html [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+      </svg>
+    </a>
+  </div>
+  <div class="container mt-5 about-us" id="about-us">
+    <div class="row">
+    <div class="col me-5">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: always use margin-right instead margin-end.

File: CogentLabs.html [hunk 9]
```diff
@@ -0,0 +1,440 @@
...
+      </div> 
+    </div>
+    </div>
+    <div class="row set-us-apart">
+      <div class="col">
+        <img src="images/image-6.png">
+      </div>
+      <div class="col">
+        <h4>What Sets Us Apart?</h4>
+        <div class="listing">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: set-us-apart-listing

File: CogentLabs.html [hunk 10]
```diff
@@ -0,0 +1,440 @@
...
+          <div class="work-list">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: set-us-apart-item

File: CogentLabs.html [hunk 11]
```diff
@@ -0,0 +1,440 @@
...
+                that runs at scale. We have helped businesses grow up to 
+                30x year over year.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-fluid mt-5 portfolio" id="portfolio">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: remove container-fluid

File: CogentLabs.html [hunk 12]
```diff
@@ -0,0 +1,440 @@
...
+    <div class="d-flex flex-column align-items-center portfolio-header">
```
Comment (Eyyan-Amir) [inline CogentLabs.html]: move this bs styling to portfolio-header

File: fontface.css [hunk 13]
```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: myFirstFont;
```
Comment (Eyyan-Amir) [inline fontface.css:2]: rename to `SF Pro Display`

File: style.css [hunk 14]
```diff
@@ -0,0 +1,316 @@
...
+}
+.navbar ul {
+  display: flex;
+  list-style-type: none;
+}
+.navbar ul a {
+  font-size: 24px;
+  font-weight: 500;
+  text-decoration: none;
+  color: black;
```
Comment (Eyyan-Amir) [inline style.css]: try to use hex color

File: style.css [hunk 15]
```diff
@@ -0,0 +1,316 @@
...
+  margin-left: 5px;
+}
+
+.about-us .row {
+  align-items: center;
+}
+.about-us h3 {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 68px;
```
Comment (Eyyan-Amir) [inline style.css]: remove line height

File: style.css [hunk 16]
```diff
@@ -0,0 +1,316 @@
...
+  font-weight: 600;
+}
+.about-us .set-us-apart {
+  margin-top: 80px;
+}
+.about-us .set-us-apart .listing .work-list {
+  display: flex;
+}
+.about-us .set-us-apart .listing .work-list .counter {
+  flex: 0 0 8%;
```
Comment (Eyyan-Amir) [inline style.css]: use px their

File: style.css [hunk 17]
```diff
@@ -0,0 +1,316 @@
...
+}
```
Comment (Eyyan-Amir) [inline style.css:209]: about-us {
.set-us-apart {
listing {}
}
}

File: style.css [hunk 18]
```diff
@@ -0,0 +1,316 @@
...
+  color: white;
+  background-color: #b624ff;
+}
+.about-us .set-us-apart .listing .work-list .content {
+  flex: 1;
+}
+.about-us .set-us-apart .listing .work-list .content h6 {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 60px;
```
Comment (Eyyan-Amir) [inline style.css]: remove

File: style.scss [hunk 19]
```diff
@@ -0,0 +1,273 @@
...
+    font-weight: 800;
+    span {
+      display: block;
+    }
+  }
+  p {
+    @include p;
+    line-height: 40px;
+    max-width: 700px;
+    margin: 0 auto auto auto;
```
Comment (Eyyan-Amir) [inline style.scss]: use margin-top

File: style.scss [hunk 20]
```diff
@@ -0,0 +1,273 @@
...
+  h1 {
+    font-size: 60px;
+    font-weight: 800;
+    span {
+      display: block;
+    }
+  }
+  p {
+    @include p;
+    line-height: 40px;
```
Comment (Eyyan-Amir) [inline style.scss]: remove

File: style.scss [hunk 21]
```diff
@@ -0,0 +1,273 @@
...
+    max-width: 700px;
+    margin: 0 auto auto auto; 
+  }
+  .btn {
+    @include button;
+    margin: 35px 0px 35px;
```
Comment (Eyyan-Amir) [inline style.scss]: margin: 35px 0px 35px; -> margin: 35px 0px;

**Review by Eyyan-Amir — CHANGES_REQUESTED** [review]

**Pull request merged**